src/mscorlib/shared/System/Diagnostics/Tracing/TraceLogging/EmptyStruct.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Diagnostics.Tracing
     /// <summary>
     /// TraceLogging: Empty struct indicating no payload data.
     /// </summary>
-    internal struct EmptyStruct
+    internal readonly struct EmptyStruct
```

Comment:
@brianrob Are you ok with marking these as readonly - will it cause issues with source sharing?

Reply:
This is also in the same boat as the other "empty" structs below.  I'm fine keeping the changes for these and fine getting rid of them.